docs/ARCHITECTURE.md

```diff
@@ -201,3 +201,67 @@ can be found under various directories such as `factory/`, `modprobe.d/`, `netwo
 `tools/`, `coccinelle/`, `.github/`, `.semaphore/`, `.mkosi/` host various
 utilities and scripts that are used by maintainers and developers. They are not
 shipped or installed.
+
+## UEFI
+The systemd project provides a UEFI bootloader, `systemd-boot`, and a stub that can be attached to a kernel
```

Comment:
I'd call this a "boot menu" btw, not a boot loader, we don't "load" anything really, we just show a menu and then call other uefi tool for each item.